README.md

```diff
@@ -218,6 +222,12 @@ error: command 'g++' failed with exit status 1
 
 This is caused by `ld` from Conda environment shadowing the system `ld`. You should use a newer version of Python that fixes this issue. The recommended Python version is 3.6.10+, 3.7.6+ and 3.8.1+.
 
+
+On Linux with ROCm support
+```bash
+.jenkins/pytorch/build.sh
+```
```

Comment:
This wouldn't work for most users, would it?

Reply:
Nope, it wouldn't.  Will fix.

Reply:
Corrected it.